apps/test/unit/javalab/redux/editorReduxTest.js

```diff
@@ -0,0 +1,41 @@
+import {expect} from '../../../util/reconfiguredChai';
+import {
+  stubRedux,
+  restoreRedux,
+  registerReducers,
+  getStore
+} from '@cdo/apps/redux';
+import javalabEditor, {
+  initialState,
+  sourceTextUpdated
+} from '@cdo/apps/javalab/redux/editorRedux';
+
+describe('javalabRedux', () => {
+  let store;
+  beforeEach(() => {
+    stubRedux();
+    registerReducers({javalabEditor});
+    store = getStore();
+  });
+  afterEach(() => restoreRedux());
+
+  it('sourceTextUpdated updates source text and retains other properties', () => {
```

Comment:
Is this test supposed to be deleted from `javalabReduxTest`?

Reply:
yep I meant to delete that and forgot, good catch!